docs/security/using-kmip.md

```diff
@@ -1,7 +1,5 @@
 # Using the Key Management Interoperability Protocol (KMIP)
 
-This feature is **technical preview** quality.
-
```

Comment:
Should we mention that as of 8.0.30 the feature is considered Generally Available?

Reply:
Added